pkg/models/shortUrl.go

```diff
@@ -0,0 +1,47 @@
+package models
+
+import (
+	"errors"
+)
+
+var (
+	ErrShortURLNotFound = errors.New("Short URL not found")
+)
+
+type ShortUrl struct {
+	Id         int64
+	OrgId      int64
+	Uid        string
```

Comment:
[Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):
```suggestion
	ID         int64
	OrgID      int64
	UID        string
```

Reply:
as in https://github.com/grafana/grafana/pull/27409#discussion_r499908517, capitalizing as recommended in the core `ShortUrl` model causes the SQL statements to be generated incorrectly, e.g. querying column `u_i_d` instead of `uid`. i updated all the others as per your recommendations though.

in general i dont love leaving this model inconsistent with all the others, but there seem to be [lots of places](https://github.com/grafana/grafana/pull/27409/files#diff-d53d40497e0faf2cae1fd2143da7d6dbR27) where the recommended conventions conflict with existing code so i'm not too worried

Reply:
I'm not sure if we want to add something like this, [ref](https://gobook.io/read/gitea.com/xorm/manual-en-US/chapter-02/4.columns.html). Not sure if it works just adding the names without data type. If we have to add the data types I think it's not worth it. 

```suggestion
	Id         int64 `xorm:"'id'"`
	OrgId      int64 `xorm:"'org_id'"`
	Uid        string `xorm:"'uid'"`
```

Reply:
Have you tried the above suggestion by @marefr @MisterSquishy?